env.js

```diff
@@ -113,6 +113,7 @@ function setStorage() {
   env.profile_collection = readENV('MONGO_PROFILE_COLLECTION', 'profile');
   env.devicestatus_collection = readENV('MONGO_DEVICESTATUS_COLLECTION', 'devicestatus');
   env.food_collection = readENV('MONGO_FOOD_COLLECTION', 'food');
+  env.azurepush_collection = readENV('MONGO_AZUREPUSH_COLLECTION', 'azurepush');
```

Comment:
I see, was thinking the registrations were for each push, but are for the phones running the apps.  A `devices` collection would be confusing with `devicestatus`, maybe we could call it `clients`, but it is more special purpose than I was thinking.  Just don't want each new integration adding a collection if we don't need to.